keyboards/lets_split/keymaps/lbibass/keymap.c

```diff
@@ -0,0 +1,200 @@
+        KC_ESC, RGB_TOG, KC_LALT, KC_LCTL, LOWER, KC_SPC, KC_SPC, _______, (CT_media), KC_VOLD, KC_VOLU, KC_MFFD),
+
+    /* Lower
+ * ,-----------------------------------------------------------------------------------.
+ * |   `  |   1  |   2  |   3  |   4  |   5  |   6  |   7  |   8  |   9  |   0  | Del  |
+ * |------+------+------+------+------+-------------+------+------+------+------+------|
+ * | Del  |  F1  |  F2  |  F3  |  F4  |  F5  |  F6  |   -  |   =  |   [  |   ]  |  \   |
+ * |------+------+------+------+------+------|------+------+------+------+------+------|
+ * |      |  F7  |  F8  |  F9  |  F10 |  F11 |  F12 |ISO # |ISO / |      |      |      |
+ * |------+------+------+------+------+------+------+------+------+------+------+------|
+ * |      |      |      |      |      |             |      | media| Vol- | Vol+ | skip |
+ * `-----------------------------------------------------------------------------------'
+ */
+    [_LOWER] = LAYOUT(
+        KC_GRV, KC_1, KC_2, KC_3, KC_4, KC_5, KC_6, KC_7, KC_8, KC_9, KC_0, KC_DEL,
+        KC_LGUI, KC_F1, KC_F2, KC_F3, KC_F4, KC_F5, KC_F6, KC_MINS, KC_EQL, KC_LBRC, KC_RBRC, KC_BSLS,
+        _______, KC_F7, KC_F8, KC_F9, KC_F10, KC_F11, KC_F12, KC_NUHS, KC_NUBS, _______, _______, _______,
+        _______, _______, _______, _______, _______, _______, _______, _______, CT_media, KC_VOLD, KC_VOLU, KC_MFFD),
+
+    /* Adjust (Lower + Raise)
+ * ,-----------------------------------------------------------------------------------.
+ * |      | Reset|      |      |      |      |      |      |      |      |      |  Del |
+ * |------+------+------+------+------+-------------+------+------+------+------+------|
+ * |      |      |      |Aud on|Audoff|AGnorm|AGswap|Qwerty|Colemk|Dvorak|      |      |
+ * |------+------+------+------+------+------|------+------+------+------+------+------|
+ * |      |      |      |      |      |      |      |      |      |      |      |      |
+ * |------+------+------+------+------+------+------+------+------+------+------+------|
+ * |      |      |      |      |      |             |      |      |      |      |      |
+ * `-----------------------------------------------------------------------------------'
+ */
+    [_ADJUST] = LAYOUT(
+        _______, RESET, _______, _______, _______, _______, _______, _______, _______, _______, _______, KC_DEL,
+        _______, _______, _______, AU_ON, AU_OFF, AG_NORM, AG_SWAP, QWERTY, WINDOWS, _______, _______, _______,
+        _______, MU_TOG, MU_MOD, _______, _______, _______, _______, _______, _______, _______, _______, _______,
+        _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______)};
+
+#ifdef AUDIO_ENABLE
+float tone_qwerty[][2]     = SONG(QWERTY_SOUND);
+float tone_colemak[][2]    = SONG(COLEMAK_SOUND);
+#endif
+
+
+
+
+void persistent_default_layer_set(uint16_t default_layer) {
```

Comment:
This function isn't needed, it's been replaced with `set_single_persistent_default_layer(#)` instead.

And it handles the audio if you set it like: 
https://github.com/qmk/qmk_firmware/blob/master/users/drashna/config.h#L8-L12
```
#define DEFAULT_LAYER_SONGS { SONG(QWERTY_SOUND), \
                                  SONG(COLEMAK_SOUND) \
                                }
```

Reply:
How should I go about configuring the set_single_persistent_default_layer(0) so that it compiles? Can't seem to find documentation on that.

Reply:
https://docs.qmk.fm/#/ref_functions?id=setting-the-persistent-default-layer